### PR TITLE
Feature/webpack setup

### PR DESCRIPTION
**Pull request by dimigor**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/PYTHON-98)

## Code reviewers

- [x] @meelros
- [ ] @BoroviyOrest
- [ ] @Kardan1123
- [x] @mizin4ik
- [x] @PetrushynskyiOleksii
- [x] @Sasha1152

### Second Level Review

- [ ] @lhalam

## Summary of change

Create basic webpack setup for react and add start page 

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Sasha1152 — DISMISSED** [review]
Good

**Review by victoria-kit — DISMISSED** [review]

**Review by PetrushynskyiOleksii — COMMENTED** [review]
Looks good

File: way_to_home/static/src/component/routeResult/routeResult.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <div className='routeResult'>
+                <p>&nbsp;A&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;00:05:20</p>
+            </div>
+            <button onClick={this.hideForm} className="hideBtnR">X</button>
+        </div>
+        )
+    }
+}
+
+export default ResultForm
```
Comment (ssOleg) [inline way_to_home/static/src/component/routeResult/routeResult.js]: Do we need to add a new empty line at the end of all files?

Comment (dimigor) [reply way_to_home/static/src/component/routeResult/routeResult.js]: Updated

**Review by PetrushynskyiOleksii — DISMISSED** [review]

**Review by lhalam — DISMISSED** [review]

**Review by Sasha1152 — APPROVED** [review]

**Review by meelros — APPROVED** [review]
Everything is good!

**Pull request merged**